timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -127,14 +204,165 @@ export default {
       return this.$store.state.meta
     },
   },
+  created() {
+    if (this.timeline) {
+      ApiClient.getSketchTimelineAnalysis(this.sketch.id, this.timeline.id)
+        .then(response => {
+          this.analyses = response.data.objects[0]
+        })
+        .catch(e => { })
+    }
+    // If no timeline was specified then loop over all of them
+    else {
+      this.sketch.timelines.forEach(timeline => {
+        ApiClient.getSketchTimelineAnalysis(this.sketch.id, timeline.id)
+          .then(response => {
+            this.analyses = this.analyses.concat(response.data.objects[0])
+          })
+          .catch(e => { })
+      })
+    }
+  },
+  mounted() {
+    this.loadSketchSigmaTags()
+    this.loadSketchTTP()
+  },
   methods: {
-    parseSigma: function(event) {
-      ApiClient.getSigmaByText(this.text)
+    problem_detector: function (search_query) {// eslint-disable-line
+      let reason = "OK"
+      if (!search_query.includes("data_type") || search_query.includes("source_name")) {
+        reason = ("No data_type or source_name defined in the rule consider add field mappings in the sigma config file")
+      }
+      if (search_query.length < 10) {
+        reason.concat("Query seems very short. It might return a lot of events and be to broad.")
+      }
+      return reason
+    },
+    templateSelected: function () {
+      // I am actually not sure what this is for, but it works.
+    },
```

Comment:
Please remove, or figure out what this is for :)

Reply:
done